src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -95,6 +95,7 @@ public RouteValueDictionary()
     /// property names are keys, and property values are the values, and copied into the dictionary.
     /// Only public instance non-index properties are considered.
     /// </remarks>
+    [RequiresUnreferencedCode("RouteValueDictionary uses reflection to generate the route values.")]
     public RouteValueDictionary(object? values)
```

Comment:
I think we might want new public API ctors for `RouteValueDictionary`.

Copying values from a `IDictionary` is fine.  Copying values via reflection from public properties seems incompatible with trimming.